Curve-finance-suite/Pool-Migration/src/agent.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Finding,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+  HandleTransaction,
+} from 'forta-agent';
+import {
+  FindingGenerator,
+  provideFunctionCallsDetectorHandler
+} from 'forta-agent-tools';
+import { utils } from 'ethers';
+
+// Pool Migrator interface
+export const PM_IFACE: utils.Interface = new utils.Interface([
+  'function migrate_to_new_pool(address _old_pool, address _new_pool, uint256 _amount) external returns (uint256)',
+]);
+
+const POOL_MIGRATION_ADDRESS: string = "0xd6930b7f661257DA36F93160149b031735237594";
+
+const createFindingGenerator = (alertId: string): FindingGenerator => {
+  return (metadata: { [key: string]: any } | undefined): Finding => {
+    return Finding.fromObject({
+      name: 'Pool Migration Finding',
+      description: 'Pool migrated to new address',
+      alertId: alertId,
+      severity: FindingSeverity.Medium,
+      type: FindingType.Unknown,
+      metadata:{
+        from: metadata?.from,
+        to: metadata?.to,
```

Comment:
Is this necessary? I think it will always be the same right? Because you are filtering by that address.

Reply:
You are correct and will update accordingly.